homeassistant/components/reolink/update.py

```diff
@@ -59,9 +61,6 @@ def installed_version(self) -> str | None:
     @property
     def latest_version(self) -> str | None:
         """Latest version available for install."""
-        if self.coordinator.data is None:
-            return None
-
```

Comment:
If the check for new firmware fails, it will raise either a `TimeoutError` or `UpdateFailed`. Both will set `last_updaate_success = False` which in turn will set the `available` property to `False`. So this shouldn't be necessary.

https://github.com/home-assistant/core/blob/e00ff548690723f0ac0a8bbb511b911c708022b2/homeassistant/components/reolink/__init__.py#L85-L91

Reply:
True, this can indeed be removed.